### ISSUE TITLE
[Feature Request]: Allow editing labels for multiple columns/items

### ISSUE THREAD

**Issue by richlv** [jasp-stats/jasp-issues#2745]
### Description

As per #2065 changing scales for multiple columns/items in one go would be useful.

**Comment by JorisGoosen** [comment]
Is this perhaps related to, if not the same as https://github.com/jasp-stats/jasp-issues/issues/2325 or https://github.com/jasp-stats/jasp-issues/issues/2530 ?

Some of the stuff there isnt implemented yet. But I guess selecting a 1 or more columns and then defining levels through an interface could work well indeed. @JohnnyDoorn ?

**Comment by richlv** [comment]
From my uninformed point of view, it seems different - it's not about re-encoding, only editing labels.
Thus the same thing as already is possible for a single column, just being able to select multiple columns and then editing labels.
Possibly with some UI hints on which values/labels already are present or are set on all selected columns, as described with the potential use of colour/bold.

Am I misunderstanding the two mentioned issues, perhaps?

**Comment by tomtomme** [comment]
@JorisGoosen 
I agree with richlv. The other requests are about recoding the values. Richlvs request is about the labels. 
But maybe that difference is not important from the developement-perspective?

**Comment by JorisGoosen** [comment]
Well, you'd have to collet all value-label pairs for all selected columns anyway and show them to the user at once. 
I was thinking of just reusing the current column-level-editor and then you can do both recoding and labelchanges at the same time. And then simply apply those labels to all of the selected columns.

**Comment by richlv** [comment]
Thanks, if I got your description suggestion right, that's what I was doing. It wasn't sufficient when sharing the file with others, though - in-file descriptions were needed.

Labels - similar, people this is shared with expect it to be self-contained and have the detail in there. Additionally, I have to agree that some analysis output is very hard to read if labels have to be manually correlated all the time.

This is data from a study that bundles multiple questionnaires, questions going up to 40 in a single questionnaire.
Some of them have multiple resulting variables (different subscales or properties) - 4, 5 etc.
The one with 40 items is reprocessed in different ways - it has 10 factors, and each factor has subitems that are expressed in confidence, congruence/not, correctness etc. While some calculation paths result in squashing, it adds more variables - and many others cannot be squashed.

On top of this, the description/label desire is user-requested, and it does seem reasonable to me - checking raw data and computed data parameters is helpful to understand analyses. So it's kinda "have to do it" situation as well :)
